src/System.Net.Security/tests/FunctionalTests/DummyTcpServer.cs

```diff
@@ -140,40 +143,25 @@ private void OnAccept(IAsyncResult result)
                 if (useSsl)
                 {
                     state = new ClientState(client, sslEncryptionPolicy);
-                    Logger.LogInformation("Server: starting SSL authentication.");
+                    _log.WriteLine("Server: starting SSL authentication.");
 
 
                     SslStream sslStream = null;
                     X509Certificate2 certificate;
+                    certificate = new X509Certificate2("DummyTcpServer.pfx");
```

Comment:
> I think that right now the Unix version incorrectly always de-prioritizes the cert with the private key (trying to match the order that Windows seemed to be writing/reading a full chain for a collection). Using the collection and finding the one which says HasPrivateKey=>true is definitely the most stable approach for a multi-cert PFX.

I didn't realize this was a multi-cert PFX. Since I'm not trying to test X509 infrastructure, I'll change the cert.

> If we have to use binary files, they need to be added to the corefx-testdata depot and not the corefx repo.

@davidsh I can't find the documentation about corefx-testdata starting at README.md... I'd like to understand more about this.

> Otherwise, I agree we should Base64 encode, etc. the needed data.

I agree: Thanks for noticing this. (We've previously agreed I'd do this in an offline discussion.)

I will create a new self-signed certificate for contoso.com and export it as Base64.